application/modules/setup/sql/032_1.5.9.sql

```diff
@@ -0,0 +1,9 @@
+# Added for versioning
+ALTER TABLE `ip_products`
```

Comment:
These need to go into the 1.6.0 sql file. Please pull the latest changes from the 1.6.0 branch